core/federated/RTI/rti.c

```diff
@@ -47,6 +47,10 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
 #include "rti_lib.h"
+
+/** Termination function */
+void termination();
```

Comment:
Instead of a forward declaration here, why not just define the function here?

Reply:
I sometimes do this when I want to avoid cluttering the beginning of the source file with "unimportant" function definitions.

Reply:
I generally want to have `main` as the first function in a file...
But I will update it so that the coding style will be the same.